docs/content/preview/reference/drivers/go/yb-pgx-reference.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Go Drivers
+linkTitle: Go Drivers
+description: Go Drivers for YSQL
+headcontent: Go Drivers for YSQL
+image: /images/section_icons/sample-data/s_s1-sampledata-3x.png
+menu:
+  preview:
+    name: Go Drivers
+    identifier: ref-yb-pgx-go-driver
+    parent: drivers
+    weight: 600
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+   <li >
+    <a href="/preview/reference/drivers/go/yb-pgx-reference/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+       YugabyteDB PGX Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="/preview/reference/drivers/go/pgx-reference/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PGX Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="/preview/reference/drivers/go/pq-reference/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PQ Driver
+    </a>
+  </li>
+
+</ul>
+
+[YugabyteDB PGX driver](https://github.com/yugabyte/pgjdbc) is a Go driver for [YSQL](/preview/api/ysql/) based on [PGX driver](https://github.com/jackc/pgx/).
+
+## Quick start
+
+Learn how to establish a connection to YugabyteDB database and begin CRUD operations using the steps from [Build a Go application](../../../../quick-start/build-apps/go/ysql-yb-pgx/).
+
+This page provides details for getting started with [YugabyteDB PGX driver](https://github.com/yugabyte/pgx) for connecting to YugabyteDB YSQL API.
+
+## Import the driver package
+
+You can import the YugabyteDB PGX driver package by adding the following import statement in your Go code.
+
+```go
+import (
+  "github.com/yugabyte/pgx/v4"
+)
+```
+
+Optionally, you can choose to import the pgxpool package instead. Refer to [Using pgxpool API](#using-pgxpool-api) to learn more.
+
+## Fundamentals
+
+Learn how to perform common tasks required for Go application development using the YugabyteDB PGX driver.
+
+### Connect to YugabyteDB database
+
+After setting up the driver, implement the Go client application that uses the driver to connect to your YugabyteDB cluster and run a query on the sample data.
+
+The YugabyteDB PGX driver allows Go programmers to connect to YugabyteDB to execute DMLs and DDLs using the PGX APIs. It also supports the standard `database/sql` package.
+
+Use the `pgx.Connect()` method or `pgxpool.Connect()` method to create a connection object for the YugabyteDB database. This can be used to perform DDLs and DMLs against the database.
+
+The driver has the following features:
+
+- It is **cluster-aware**, which eliminates the need for an external load balancer.
+
+Connections are distributed across all the [YB-Tservers](/preview/architecture/concepts/yb-tserver/) in the cluster, irrespective of their placements.
+
+The following table describes the connection parameters required to connect to the YugabyteDB database with **Uniform load balancing**.
```

Comment:
Top 5 params are the standard ones. Only `load_balance` is the one which enables uniform (or cluster-aware) load balancing.